### DISCUSSION TITLE
Pynecone with Django

### DISCUSSION THREAD

**Discussion by saurabhwadekar** [reflex-dev/reflex#475] [General]
I am thinking to use pynecon in frontend and Django in backend is it possible

**Comment by Lendemor** [comment]
You can't really remove the pynecone backend, but it is possible for your pynecone backend to send request to a Django API if that's what you want.

**Comment by saurabhwadekar** [comment]
but i don't want to host two servers

**Reply by Lendemor** [reply]
Then run your Django on same server but different port.

**Reply by saurabhwadekar** [reply]
**thanks bro**
